blakes24/capstone1
# Add dive journal feature

### What does this PR do?
- This PR allows users to save, edit, and delete a dive site on their dive journal
### Description of Task to be completed
- A user clicks a button to save a site to their journal and add additional info
- A user clicks a button on the details page to edit 
- A user clicks a button on the details page to delete the site
### How should this be manually tested?
- Clone the repository
- Sign in
- Search for a site to add to your journal, click the save button and fill out the form
- Go to dive journal and click a site to see and edit details
- Click the delete button to remove the site
